packages/mantle/src/components/combobox/combobox.tsx

```diff
@@ -181,7 +181,6 @@ const ComboboxItem = forwardRef<
 			children,
 			className,
 			focusOnHover = true,
-			setValueOnClick = true,
```

Comment:
do you want to do this or pass it to the props of the primitive below?

Reply:
`true` is the [default in Ariakit](https://ariakit.org/reference/combobox-item#setvalueonclick), do we need to specify it on our side?